test/external_project/seqan3_submodule_find_package/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ include (../find-package-diagnostics.cmake)
 # ---
 
 # add seqan3 to search path
-list (APPEND CMAKE_PREFIX_PATH "${CMAKE_CURRENT_SOURCE_DIR}/submodules/seqan3/build_system") # change this value
+list (APPEND CMAKE_PREFIX_PATH "${CMAKE_CURRENT_SOURCE_DIR}/submodules/seqan3/cmake") # change this value
```

Comment:
Confusing comment, why should this value be changed?

Reply:
It might have been a snippet in the tutorial, but it isn't anymore.